table.cc

```diff
@@ -1089,7 +1089,11 @@ void table::set_metrics() {
                 ms::make_gauge("live_disk_space", ms::description("Live disk space used"), _stats.live_disk_space_used)(cf)(ks),
                 ms::make_gauge("total_disk_space", ms::description("Total disk space used"), _stats.total_disk_space_used)(cf)(ks),
                 ms::make_gauge("live_sstable", ms::description("Live sstable count"), _stats.live_sstable_count)(cf)(ks),
-                ms::make_gauge("pending_compaction", ms::description("Estimated number of compactions pending for this column family"), _stats.pending_compactions)(cf)(ks)
+                ms::make_gauge("pending_compaction", ms::description("Estimated number of compactions pending for this column family"), _stats.pending_compactions)(cf)(ks),
+                ms::make_gauge("partition_creations", ms::description("Number of times a write operation created a new partition"), _stats.partition_creations)(cf)(ks),
+                ms::make_gauge("partition_reuses", ms::description("Number of times a write operation was issued in an existing partition"), _stats.partition_reuses)(cf)(ks),
+                ms::make_gauge("row_creations", ms::description("Number of times a new row was created"), _stats.row_creations)(cf)(ks),
+                ms::make_gauge("row_overwrites", ms::description("Number of times a row was updated"), _stats.row_overwrites)(cf)(ks)
         });
```

Comment:
Maybe better is rows (with the sum) and row_overwrites (similar for partitions). This gives you a single counter for partition and row writes which is the most interesting thing, and a secondary counter for overwrites.

Reply:
OK. Changed those counters to report total writes and overwrites (`memtable_partition_writes` + `memtable_partition_hits`, `memtable_row_writes` + `memtable_row_hits`).